packages/sync-actions/src/product-types-actions.js

```diff
@@ -188,13 +186,13 @@ export function actionsMapAttributes(
             attributeName: extractedPairs.oldObj.name,
           }))
         )
-      } else if (diffValue.values) {
+      } else if (diffValue.type.values) {
         actions.push(
           ...actionsMapEnums(
             extractedPairs.oldObj.type.name,
-            diffValue,
-            extractedPairs.oldObj,
-            extractedPairs.newObj
+            diffValue.type,
```

Comment:
This is the actual fix, correct?

Reply:
yes this is the actual fix.
Also, regarding your concern about test, I did adjust `createAttributeDefinition` to match the right shape.